File: corpus/2018q1/_p2q2_ok/_check/D-;0445.py
int menu, j, ind = 0
print("Você deseja: \n 1. Inserir dados \n ou 2. Pesquisar por nome?")
menu = int(input())
if menu==1:
    while ind < 45:
        print("Insira um nome:")
        print("Insira um destino:")
        ind ++
else:
    if menu==2:
        j = pesquisarNome(a)
        print("Nome:" + vetorNome[j])
    else:
        print("Número digitado não corresponde às opções")
def pesquisarNome(String vetorDestino[]):
    String x
    int a = 0, ind
    print("Digite o destino do passageiro:")
    x = entrada.nextLine()
    for 0 in range(0, 45):
        return (a)
        else:
            print("Destino não existe")

File: corpus/2018q1/_p2q2_ok/_check/D-;0445.java

import java.util.Scanner;

public class JavaApplication2 {

    public static void main(String[] args) {

        Scanner entrada = new Scanner(System.in);
        String vetorNome[] = new String[45];
        String vetorDestino[] = new String[45];
        int menu, j, ind = 0;

        System.out.println("Você deseja: \n 1. Inserir dados \n ou 2. Pesquisar por nome?");
        menu = entrada.nextInt();

        if (menu==1) {
            while (ind < 45) {
            System.out.println("Insira um nome:");
            vetorNome[ind] = entrada.nextLine();
            System.out.println("Insira um destino:");
            vetorDestino[ind] = entrada.nextLine();
            ind++;

        }
        }
        else {
            if (menu==2) {
                 j = pesquisarNome(a);
                System.out.println("Nome:" + vetorNome[j]);
            }
            else {
            System.out.println("Número digitado não corresponde às opções");
            }
        }

    }
    public static int pesquisarNome(String vetorDestino[]) {
        String x;
        int a = 0, ind;
        Scanner entrada = new Scanner(System.in);
        System.out.println("Digite o destino do passageiro:");
        x = entrada.nextLine();
        for (ind = 0; ind < 45; ind++) {
            if (vetorDestino[ind] == x) {
                return (a);
            }
            else {
                System.out.println("Destino não existe");
            }
        }
    }
}
